### PR TITLE
Sf3/PHP7 update

### PR DESCRIPTION
**Pull request by kaltun**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,16 +11,16 @@
     ],
     "require": {
         "php": ">=5.4",
```
Comment (N3m1s) [inline composer.json]: The code uses `::class` this is only available from PHP 5.5

File: composer.json [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
         "doctrine/orm": "~2.5",
         "doctrine/doctrine-bundle": "~1.5",
         "sensio/framework-extra-bundle": "~3.0",
 
-        "gedmo/doctrine-extensions": "dev-master",
+        "gedmo/doctrine-extensions": "2.4.*",
         "stfalcon/tinymce-bundle": "~1.1|~2.1",
 
-        "nfq-rho/admin-bundle": "~0.5"
+        "nfq-rho/admin-bundle": "dev-sf3_update"
```
Comment (N3m1s) [inline composer.json]: Wrong branch

File: composer.json [hunk 3]
```diff
@@ -11,16 +11,16 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/symfony": "~2.6",
+        "symfony/symfony": "~3",
```
Comment (N3m1s) [inline composer.json]: Looking at the changes made, I think this bundle will work with Symfony versions 2.7, 2.8, too. So the rule should be something like `"symfony/symfony": "~2.7|~3.0"`

**Review by N3m1s — CHANGES_REQUESTED** [review]

File: Resources/config/routing.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 _cms_admin:
-    resource: @NfqCmsPageBundle/Resources/config/routing_admin.yml
+    resource: "@NfqCmsPageBundle/Resources/config/routing_admin.yml"
     prefix: /admin/cms-pages
     options: { i18n: false }
 
 _cms_front:
-    resource: @NfqCmsPageBundle/Resources/config/routing_front.yml
+    resource: "@NfqCmsPageBundle/Resources/config/routing_front.yml"
```
Comment (N3m1s) [inline Resources/config/routing.yml]: Would prefer single quotes

File: Resources/config/services.yml [hunk 5]
```diff
@@ -30,6 +30,11 @@ services:
             - [setContainer, [ '@service_container' ]]
             - [setConfig, [ '%nfq_cmspage.types%' ]]
 
+    nfq_cmspage.cms_page_type:
```
Comment (N3m1s) [inline Resources/config/services.yml]: Is this service used anywhere?

**Review by N3m1s — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (kaltun) [reply composer.json]: As I understand, it should be changed when new admin-bundle version will be released.

File: composer.json (hunk 1 above)
Comment (kaltun) [reply composer.json]: fixed

File: composer.json (hunk 3 above)
Comment (kaltun) [reply composer.json]: fixed

File: Resources/config/routing.yml (hunk 4 above)
Comment (kaltun) [reply Resources/config/routing.yml]: changed

File: Resources/config/services.yml (hunk 5 above)
Comment (kaltun) [reply Resources/config/services.yml]: You're right, it's not used anywhere. Removed it.

**Review by N3m1s — APPROVED** [review]

**Pull request merged**